go.mod

```diff
@@ -4,7 +4,7 @@ go 1.13
 
 require (
 	cloud.google.com/go v0.45.1
-	github.com/crossplane/crossplane v0.9.0
+	github.com/crossplane/crossplane v0.10.0-rc.0.20200408163913-aed24e6eb928
```

Comment:
this looks to be the same value as what was done in provider-aws:
https://github.com/crossplane/provider-aws/commit/a8800010d59f3308e52d3d15ece39f91a2b6d180#diff-37aff102a57d3d7b797f152915a6dc16